types/angular/jqlite.d.ts

```diff
@@ -25,7 +25,7 @@
 
 // Definitions copied from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0480c5ec87fab41aa23047a02b27f0ea71aaf975/types/jquery/v2/index.d.ts
 
-interface JQuery {
+interface JQuery<TElement extends Node = HTMLElement> {
```

Comment:
You'll need to remove the generic type parameter from JQuery. I forgot about this on the first review but @types/angular only has TypeScript Version: 2.3 for tests. One of the big reasons @types/angular no longer depends on @types/jquery is because the introduction of default type parameters broke a lot of builds.